frontend/src/views/HomeView.vue

```diff
@@ -2,16 +2,66 @@
   <div class="mt-5">
     <SloganComponent />
     <div class="mt-5">
-      <DropToStartField />
+      <DropToStartField @dropFilesHandler="dropFilesHandler" />
     </div>
     <div class="mt-5">
       <FrequentlyAskedQuestions />
     </div>
   </div>
+  <div v-if="hasErorr" class="alert alert-danger alert-dismissible" role="alert">
+    <div>{{ errorMessage }}</div>
+    <button type="button" @click="dismissAlert" class="btn-close" aria-label="Close"></button>
+  </div>
 </template>
 
 <script setup lang="ts">
 import SloganComponent from '@/components/SloganComponent.vue'
 import DropToStartField from '@/components/DropToStartField.vue'
 import FrequentlyAskedQuestions from '@/components/FrequentlyAskedQuestions.vue'
+import { ref } from 'vue'
+import axios from 'axios'
+import router from '@/router'
+
+const hasErorr = ref(false)
+const errorMessage = ref('')
+
+function dismissAlert() {
+  hasErorr.value = false
+}
+
+async function dropFilesHandler(event: DragEvent) {
+  if (event.dataTransfer?.files == undefined) {
+    return
+  }
+  const files = event.dataTransfer.files
+  axios
+    .post(`${import.meta.env.VITE_API_URL}/users`)
+    .then(() => {
+      const data = {
+        name: 'First Project'
+      }
+      return axios.post(`${import.meta.env.VITE_API_URL}/projects`, data)
+    })
+    .then((response) => {
+      const project_id = response.data.id
```

Comment:
Will probably end up in a separate self-contained object but not required at the moment.
One important thing though is that you need to expose this list of files otherwise you won't be able to query it to check for file names, progress or status. Not sure if you want a custom structure as suggested in [the other PR](https://github.com/openzim/nautilus-webui/pull/29#issuecomment-1650322960) or if you can easily and reliably retrieve it all from the xhr.

Reply:
We can create an interface here and then specify what format the data received is in. This data type can also be used later. 
About file names, progress or status, etc. We will rely on the data provided by the server.

Reply:
> About file names, progress or status, etc. We will rely on the data provided by the server.

No, the upload will take a long time to complete and you won't have a response by the server until it's completed. You thus need to list the files **bing uploaded** in the UI, relying on JS-sourced data for filename and size (you can update once uploaded)